code/game/machinery/harvester.dm

```diff
@@ -154,7 +154,7 @@
 		say("Subject has been successfully harvested.")
 		playsound(src, 'sound/machines/microwave/microwave-end.ogg', 100, FALSE)
 
-/obj/machinery/harvester/screwdriver_act(mob/living/user, obj/item/I)
+/obj/machinery/harvester/screwdriver_act(mob/living/user, obj/item/some_screwdriver)
```

Comment:
what is the var name on the root of `screwdriver_act()`? if it is still `obj/item/I`, please leave it as such so we don't fuck up passing along vars. if it is something else, change it to that.